docs/conf.py

```diff
@@ -156,5 +156,6 @@
 # -- Generate POSIX API list ----------------------------------------------
 
 import subprocess
+subprocess.check_output(['sudo', 'apt', 'install', 'cscope'])
```

Comment:
Don't install `csope` from scripts. Instead, mention it carefully in top-level documentation.

Reply:
Yes, in fe8f18949 there is a prebuild `cscope`, this is for readthedocs to build the docs.